example/svrg_module/linear_regression/README.md

```diff
@@ -0,0 +1,7 @@
+### Linear Regression using SVRGModule on YearPredictionMSD dataset
+
+YearPredictionMSD: dataset contains over 400,000 samples with 90 features. Please uncomment data downloading script from
+data_reader.py to download the data. 
+
+This example trains a linear regression model using SVRGModule. Logs of the training results can be found in 
+experiments.log
```

Comment:
where is experiments.log?

Reply:
It will be generated when runs the train script.